src/compas_rhino/geometry/surfaces/surface.py

```diff
@@ -189,7 +189,8 @@ def curvature_at(self, u, v):
         :class:`~compas.geometry.Vector`
 
         """
-        vector = self.rhino_surface.CurvatureAt(u, v)
+        surface_curvature = self.rhino_surface.CurvatureAt(u, v)
+        vector = surface_curvature.Direction()
```

Comment:
I get an error with this, because the `Direction` method needs an `int direction` parameter (0 or 1):
![image](https://user-images.githubusercontent.com/933277/193696589-28b4c7e1-7f23-4200-befc-955b120aa010.png)

Reply:
Correct, I forgot about this. Apparently, with 0 it returns the maximum principal curvature direction and with 1 it returns the minimum principal curvature direction. My guess is that the maximum could be generally more useful than the minimum.
![image](https://user-images.githubusercontent.com/91247922/193776328-b68f0df1-7f3e-4792-a239-148f4cc9416b.png)

Reply:
Yes, I think so too :)

Reply:
I updated the code passing a 0 argument